routes/api/v1/dashboards.js

```diff
@@ -0,0 +1,69 @@
+const Dashboard = require('../../../models/dashboards.js')
+const { Widget } = require('../../../models/widgets.js')
+
+const express = require('express')
+const router = express.Router()
+
+router.get('/', (request, response, next) => {
+  Dashboard.find().exec()
+    .then( dashboard => response.json( dashboard ) )
+    .catch( error => next( error ) )
+})
+
+router.post('/', (request, response, next) => {
+  Dashboard.create( request.body )
```

Comment:
Did a little research and found http://mongoosejs.com/docs/api.html#model_Model.create

The create method returns a promise, so we don't have to add ```.exec()```. We've also tested it using Postman and it's been working!